x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/conditional/CaseTests.java

```diff
@@ -44,6 +44,7 @@ public CaseTests(@Name("TestCase") Supplier<TestCaseSupplier.TestCase> testCaseS
      */
     @ParametersFactory
     public static Iterable<Object[]> parameters() {
+        // TODO this needs lots of stuff flipped to parameters
```

Comment:
What do you mean with this?

Reply:
There's a ton of methods testing things that all the other scalars use parameterized tests to check.